src/custom/state/claim/hooks/index.ts

```diff
@@ -20,8 +20,12 @@ import { isAddress } from 'utils'
 
 import { getClaimKey, getClaimsRepoPath, transformRepoClaimsToUserClaims } from 'state/claim/hooks/utils'
 import { SupportedChainId } from 'constants/chains'
+import { registerOnWindow } from 'utils/misc'
+import mockData, { MOCK_INDICES } from './mocks/claimData'
+import { getIndexes } from './utils'
+import { useAllClaimingTransactionIndices } from 'state/enhancedTransactions/hooks'
 
-export { useUserClaimData } from '@src/state/claim/hooks'
+export { useUserClaimData, useUserUnclaimedAmount } from '@src/state/claim/hooks'
```

Comment:
Hmm, are you sure you want to export the original? Won't that break anything?

Reply:
yes this shouldn't be here @nenadV91